### DISCUSSION TITLE
jQueryについて

### DISCUSSION THREAD

**Discussion by mitukawa** [zemi-team2/sake#45] [Q&A]
# 内容
画像をサーバーにアップロードする処理を実装する際に`jQuery`で`Ajax`を使う必要があるのですが、`jQuery`を使う場合は`CDN`で読み込む形で大丈夫でしょうか？

やりたいことは、非同期通信で、画像(文字列化したもの)をserverにpostすることです。

# 関連Issue
#37

**Comment by watagit** [comment]
個人的には JS を使って API 通信する際には`fetch`メソッドか`axios`ライブラリを使うことが多いです。
最終的に処理が実現できさえすればそこまで差はないと思うので好みなやり方で実装してもらっていいと思います〜！

参考資料
- [fetch](https://developer.mozilla.org/ja/docs/Web/API/Fetch_API/Using_Fetch)
- [axios](https://github.com/axios/axios)
